packages/workbox-build/src/entry-points/options/defaults.js

```diff
@@ -17,10 +17,8 @@
 module.exports = {
   globFollow: true,
   globIgnores: ['**/node_modules/**/*'],
-  globPatterns: ['**/*.{js,css,html}'],
+  globPatterns: [],
```

Comment:
could we keep this default and give the webpack plugin a different default?

I'm guesisng this makes the webpack plugin require no config and requires config in webpack build.

Reply:
There's always going to be config needed in the `workbox-build` (I think that's what you're asking about), because we don't know what the `globDirectory` should be.

So yes, this change makes it so that you need to explicitly configure `globPatterns` as well as `globDirectory` when using `workbox-build` directly.

It sounds like you're not in favor of that, so I'll undo that part of this PR and continue overriding `globPatterns` to `[]` when needed, instead of the other way around.